src/wallet.js

```diff
@@ -2035,6 +2035,21 @@ const wallet = {
     const key = hdpubkey.derive(index);
     return key.publicKey.toBuffer();
   },
+
+  /**
+   * Get xpubkey from storage xpriv (assumes PIN is correct)
+   *
+   * @param {String} pin User PIN used to encrypt xpriv on storage
+   *
+   * @return {String} Wallt xpubkey
+   */
+  getXPubFromPin(pin) {
```

Comment:
I'd name it `getXPubKeyFromXPrivKey(int)`. What do you think?

Reply:
Changed!